mod/metrics/spec/lib/formula/javascript/script_value_type.test.coffee.off

```diff
@@ -0,0 +1,9 @@
+#$.extend global, require('../../../../lib/javascript/script_value_type.js.coffee')
+
+#test 'return true if it returns an array with the properties corresponding to "Number" and "Money"', =>
+#  expect(getPropertiesValueType('Number')).toEqual(['Unit','Range'])
+#  expect(getPropertiesValueType('Money')).toEqual(['Unit','Range'])
+
+#test 'return true if it returns an array with the properties corresponding to "Category" and "Multi-Category"', =>
+#  expect(getPropertiesValueType('Category')).toEqual(['Options'])
+#  expect(getPropertiesValueType('Multi-Category')).toEqual(['Options'])
```

Comment:
Why is this not active?

Reply:
I'm  not sure.  Daniel wrote it and then commented it.